shared/insideResinOS/2.x.md

```diff
@@ -20,7 +20,7 @@ ResinOS currently comes in 3 different variants all built from the same source b
 |:------------:|:------------:|:------------:|
 | 2.0.0+rev2   | production   | This is the production version of the resin.io managed OS. This is the OS you should use for any production fleet deployments |
 | 2.0.0+rev2-dev | development | This version the development version of the above and should be used when you are developing a new application and want to use the fast [local mode][local-mode-link] workflow. This variant should never be used in production. |
-| 2.0.0+rev2-standalone | standalone | This version is an unmanaged version of resinOS, it doesn't have the resin.io supervisor and does not connect to the management console. Use this variant if you want a stable linux OS to run Docker on. |
+| 2.0.0+rev2.dev | standalone | This version is an unmanaged version of resinOS, it doesn't have the resin.io supervisor and does not connect to the management console. Use this variant if you want a stable linux OS to run Docker on. |
```

Comment:
Hmmm, so the `development` variants have the same version-name as the `standalone` variants? Is that a bit confusing, or is it just in preparation for the upcoming manged + unmanaged unification that I've heard people mentioning?

Reply:
@lurch There's a slight difference—the dev versions use -dev, and the unmanaged versions use .dev